nexus/types/src/external_api/params.rs

```diff
@@ -947,6 +947,13 @@ pub enum ExternalIpCreate {
     Floating { floating_ip_name: Name },
 }
 
+#[derive(Clone, Debug, Deserialize, Serialize, JsonSchema)]
+#[serde(tag = "type", rename_all = "snake_case")]
+pub enum ExternalIpDelete {
+    Ephemeral,
+    Floating { floating_ip_name: Name },
+}
```

Comment:
Why aren't we detaching by id?

Reply:
This is mostly symmetrical to `ExternalIpCreate`, which is used in instance creation as well as IP attach. I'd prefer to specify a `NameOrId` in both structs (for both the target floating IP and ephemeral IP pool), but I'm not sure what the policy is for a breaking change there.

Reply:
I think the policy is BREAK THE API. We have basically zero consumers so now is the time to get the API right.

Reply:
Yeah, we need to fix the other one, not make this one equally bad